spec/features/index_request_spec.rb

```diff
@@ -33,4 +33,11 @@
     page.all(".itemname").each { |x| names << x.text }
     expect(names).to match_array(names.sort)
   end
+  
+  it "should not show items where listed is false" do
+    Item.create!(name: "Backpack", price: 5.00, image: "pack.jpg", description: "Holds things.", listed: false, status: "unavailable", deliverable: true, user_id:2)
```

Comment:
be careful representing images as string URLs - I don't think it really matters in this case, but it could cause problems in the future

Reply:
actually... can you verify this works after my latest push (deleting image from seeds)?

Reply:
good point- would it be better practice to just not set the image if it isn't relevant to the test?

Reply:
I think for now, yes. An image is an optional arg